build-support/bin/BUILD

```diff
@@ -52,6 +52,16 @@ python_binary(
   tags = {'type_checked'},
 )
 
+python_binary(
```

Comment:
doesn't seem like this needs to be a separate binary.

Reply:
We treat each of the `build-support` scripts as a `python_binary`. Most of them, you can directly invoke via `./build-support/bin/my_script.py`. Some of them, like this, must be invoked via `./pants run`. We consistently mark them both to allow `./pants run` and so that our githook runs over this code.